<div class="center">
    <h1 class="title">A time-course analysis of the oligodendrocyte translatome in response to spinal cord injury</h1>
</div>

<div class="one-line">
    <div class="center">
        <img src="../../assets/flow.png" width="250" class="side-image" />
    </div>

    <div class="center home-center">
        <br>
        <br>
    
        <button mat-raised-button routerLink="/search">Explore OL Translatome Dataset</button>
    
        <br>
        <br>
        <br>
        <br>
    
        <!-- <p class="description">This project utilizes RiboTag technology to isolate oligodendrocyte-specific, ribosome-associated mRNAs to investigate changes in the oligodendrocyte translatome in the spinal cord before and after progressive stages of a moderate thoracic spinal cord contusion injury. This dataset provides information about how oligodendrocytes respond to an injury at the translational level (ribosome-associated mRNAs) to identify novel mechanisms by which oligodendrocytes help or hinder the healing process after a devastating injury.</p>
    
        <br>
        <br>
        <br>
    
        <p class="footnote">Translatome: transcriptomics of isolated ribosome-associated mRNAs. These are considered “actively translating”.</p> -->

        <p class="description">Mature oligodendrocyte (OL) translatomes were isolated from the contused spinal cord tissue of OL-Ribotag mice at various times after moderate contusive SCI at the T9 level. mRNA expression changes were then determined using RNASeq. The presented database enables visualization of mRNA expression changes in OLs and all spinal cord cells as determined using DESeq2. It also contains results of the DESeq2 interaction analysis (2x2 or 2x4) to compare differential OL enrichment of gene expression after SCI. The database provides an unprecedented insight into OL response to SCI.</p>

        <br>
        <br>

        <p style="font-size: small; margin-left: 5%; margin-right: 5%;">All animal procedures were performed in strict accordance with the Public Health Service Policy on Humane Care and Use of Laboratory Animals, <i>Guide for the Care and Use of Laboratory Animals</i> (Institute of Laboratory Animal Resources, National Research Council, 2011), and adhered to NIH guidelines on the use of experimental animals. Animal procedures were approved by the University of Louisville Institutional Animal Care and Use (IACUC) and Institutional Biosafety (IBC) Committees. OL-Ribotag mice (Plp-Cre<sup>ERT2+/wt</sup>:Rpl22<sup>fl(STOP)fl-HA/wt</sup>) were induced with 1 mg tamoxifen (20 mg/mL in sunflower oil) i.p. daily for 8 days as previously to selectively label OL ribosomes. Female OL-Ribotag mice were used for SCI at 8-12 weeks of age, 3 weeks after completion of the induction treatment. T9 moderate contusive SCI was performed using the IH impactor (50 kdyn force/400-600 μm displacement, Infinite Horizons, Lexington, KY) as previously described (PMID: 32848194).  Five mm portions of the spinal cord spanning the injury epicenter were dissected and flash frozen in liquid nitrogen for Ribotag OL polysome purification that was performed as previously described (PMID: 31216392). Input polysome-associated mRNAs (total spinal cord mRNA) or immune-purified OL polysome mRNAs were isolated. mRNA library preparation and RNA sequencing on the Illumina NextSeq 500 platform followed standard procedures. Analysis of sequenced RNA was performed by the Kentucky Biomedical Research Infrastructure Network (KBRIN) Bioinformatics Core. <i>After</i> Fast QC quality control, the reads were directly aligned to the Mus musculus reference genome assembly (mm10.fa) using STAR (version 2.6) followed by raw read count (HTSeq version 0.10.0). Raw read count/sample ranged from 36,812,296 to 39,546,186 (average 37,495,614). At least 96.56% reads were aligned to the mouse reference genome (average read alignment 98.06%). Naïve, uninjured mice (C57Bl6) were used as controls. DESeq2 was used to analyze the differential gene expression by computing expected read counts and log2 fold changes based on a negative binomial regression model. The DESeq2 interaction module was used to analyze differential OL enrichment after SCI as defined by delta Log<sub>2</sub>FC(OL/total)=Log<sub>2</sub>FC(OL/total)_SCI – Log<sub>2</sub>FC(OL/total)_naive. In this analysis, OL translatome- and total spinal cord RNA samples from the same tissue were paired for determination of differential OL enrichment using binomial regression model. To identify genes with an RNA origin-specific effect (OL vs. total) at one or more time points, the full regression model was compared to a reduced model using a likelihood ratio test. Group-specific effects were then identified at individual timepoints using a Wald test for significance.</p>

    </div>

    <div class="center">
        <img src="../../assets/staining.png" width="250" class="side-image" />
    </div>
</div>


